### ISSUE TITLE
Expound cannot handle js/NaN

### ISSUE THREAD

**Issue by vemv** [bhb/expound#48]
Hi!

I found the following issue, where spec can handle NaN but Expound not:

`(spec/conform (constantly false) js/NaN)` -> `:cljs.spec.alpha/invalid`

---

`(expound/expound (constantly false) js/NaN)`:

```
{:message "Cannot convert path. This can be caused by using conformers to transform values, which is not supported in Expound", :data {:form NaN, :val NaN, :in [], :in' []}}
```

Thanks for this library, we've been enjoying it at work for quite a few months now!

Cheers - Victor

**Comment by bhb** [comment]
@vemv Thanks very much for reporting this issue! I'll take a look soon.

**Comment by bhb** [comment]
@vemv Thanks again for reporting this bug. I've put a fix in master and pushed 0.3.4-SNAPSHOT to clojars if you want to try it. I should be putting out a bug fix release with this and a few other bug fixes sometime in the next week.

**Comment by vemv** [comment]
Confirming it works! Thank you! 🍻

**Comment by bhb** [comment]
I've just released 0.3.4, which includes this bug fix.
